### PR TITLE
Debug auto orienter

### PR DESCRIPTION
**Pull request by ralatsdc**

### CODE REVIEW

**Review by robotastic — COMMENTED** [review]
looks good - just the bashrc file

File: .bashrc
[the remark this answers is not in the document]
Comment (ralatsdc) [reply .bashrc:1]: I agree. Included only to simplify debugging.

**Comment by robotastic** [comment]
Looks good enough - Question on setting Yaw, Pitch, Roll via the MQTT config message. It looks like it was removed and there is this comment:
`Cannot set tripod yaw, pitch, and roll because of side effects`
What is the side effect?

Is there still a way to manually set the Yaw Pitch Roll via MQTT? It is helpful to have a way to do some live manual adjustment. Is this still possible?

**Comment by ralatsdc** [comment]
The orientation message sets yaw, pitch, and roll. The side effect is computing the coordinate transformations, which the orientation message callback already does.

**Pull request merged**